internal/cli/kraft/cloud/compose/up/up.go

```diff
@@ -173,6 +177,45 @@ func Up(ctx context.Context, opts *UpOptions, args ...string) error {
 			return fmt.Errorf("service '%s' not found", serviceName)
 		}
 
+		var (
+			userPkgName     string
+			officialPkgName string
+		)
+
+		if service.Image != "" {
+			if !strings.Contains(service.Image, ":") {
+				service.Image += ":latest"
+			}
+
+			userPkgName = fmt.Sprintf(
+				"%s/%s",
+				userName,
+				strings.ReplaceAll(service.Image, "_", "-"),
+			)
+			officialPkgName = strings.ReplaceAll(service.Image, "_", "-")
+		} else {
+			userPkgName = fmt.Sprintf(
+				"%s/%s-%s:latest",
```

Comment:
This is not prefixed with `index.unikraft.io`?

Reply:
The API does not like this. Additionally, `index.unikraft.io` (+`/official`) is stripped in the next `create.Create` method anyway.

Reply:
okay